package.json

```diff
@@ -37,12 +37,12 @@
   "dependencies": {
     "@actions/core": "^1.5.0",
     "@actions/exec": "^1.1.0",
-    "@actions/github": "^4.0.0",
-    "@actions/glob": "^0.1.2",
+    "@actions/github": "^5.0.0",
+    "@actions/glob": "^0.2.0",
```

Comment:
Why didn't we move to `0.2.0` in https://github.com/actions/github-script/pull/192?

Reply:
I ran `npm i @actions/glob` to create #192, which I thought would bump minor versions but I think `0.y.z` has special meaning in semver so it did not bump to the "latest minor version".

>Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

https://semver.org/#spec-item-4